src/index.js

```diff
@@ -25,6 +27,12 @@ AIOCommand.run = async (argv, opts) => {
   // ||  module.parent && module.parent.parent && module.parent.parent.filename
   const config = await Config.load(opts || __dirname)
 
+  // Check Node.js version
+  const nodeVersion = process.version
+  if (!semver.satisfies(nodeVersion, config.pjson.engines.node)) {
```

Comment:
Is it too geeky to output this?
`Supported versions are ${config.pjson.engines.node}.`
Then we would just have to update package.json and not look at this again :)

Reply:
I thought about that. But wasn't sure if its ok to show the versions from package json. Made the change though
<img width="920" alt="Screenshot 2025-04-15 at 6 58 24 PM" src="https://github.com/user-attachments/assets/60f0b552-6354-4a19-b8d7-81f683c5b5b4" />